### PR TITLE
Updated to use the new itc api

### PR DESCRIPTION
**Pull request by cquiroz**
This compiles but I'm not sure how to test it

### CODE REVIEW

**Comment by cquiroz** [comment]
Besides the new API, the new ITC server brings a much faster algorithm when a s/n@wavelength parameter is specified

File: flake.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          overlays = [ typelevel-nix.overlay scala-cli-overlay ];
+        };
+      in {
+        devShell = pkgs.devshell.mkShell {
+          imports = [ typelevel-nix.typelevelShell ];
+          packages = [
+            pkgs.nodePackages.graphqurl
+            pkgs.docker
+            pkgs.docker-compose
+            pkgs.postgresql
```
Comment (tpolecat) [inline flake.nix:24]: Seems like you shouldn't need postgres.

**Review by tpolecat — APPROVED** [review]
Seems fine. 👍🏻

File: flake.nix (hunk 1 above)
Comment (cquiroz) [reply flake.nix:24]: This is to get the `psql` client

**Pull request merged**